MessengerProj/src/main/res/values/strings.xml

```diff
@@ -110,27 +110,27 @@
     <!--cache view-->
     <string name="CacheSettings">Storage</string>
     <string name="KeepMedia">Keep Media</string>
-    <string name="KeepMediaInfo">Photos, videos and other files from cloud chats that you have <![CDATA[<b>not accessed</b>]]> during this period will be removed from this device to save disk space.</string>
+    <string name="KeepMediaInfo">Photos, videos and other files from cloud chats you have <![CDATA[<b>yet to access</b>]]> during this period will be removed from this device to save disk space.</string>
     <string name="KeepMediaForever">Forever</string>
     <!--passcode view-->
-    <string name="Passcode">Passcode Lock</string>
-    <string name="ChangePasscode">Change Passcode</string>
-    <string name="ChangePasscodeInfo">When you set up an additional passcode, a lock icon will appear on the chats page. Tap it to lock the app.\n\nNote: if you forget the passcode, you\'ll need to delete and reinstall the app.</string>
+    <string name="Passcode">App Lock</string>
+    <string name="ChangePasscode">Change app lock</string>
+    <string name="ChangePasscodeInfo">When you set up an app lock, a lock icon will appear atop the chat page. Tap it to lock the app.\n\nNote: Should you forget it, you\'ll need to delete and reinstall the app.</string>
```

Comment:
My preference is to keep this:

\n\nNote: Should you forget **the passcode**, ...